app/models/user.rb

```diff
@@ -3,4 +3,7 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+  validates :bio, presence: true, length: { minimum: 5 }
```

Comment:
Looks great Liz! One thing though, is the bio minimum 5 words or 5 characters?

Reply:
Great point Tom! I think perhaps minimum 5 words is the best way forward here?

Reply:
Complete agree, great idea. I'm happy to merge